config.toml

```diff
@@ -4,9 +4,52 @@ defaultContentLanguageInSubdir = true
 baseURL = "https://solnascente.eu/"
 languageCode = "en-us"
 title = "Sol Nascente"
-theme = "hugo-now-ui"
+theme = ["hugo-now-ui"]
 staticDir = ["static"]
 
+[menu]
+
+ [[menu.main]]
+    identifier = "market"
+    name = "market landscape"
```

Comment:
These have to be translatable & translated.

Reply:
Done.